.swiftpm/Authentication.xctestplan

```diff
@@ -0,0 +1,24 @@
+{
+  "configurations" : [
+    {
+      "id" : "30BFE1C1-BEC8-4148-AE9B-5ED6C9F1DE90",
+      "name" : "Test Scheme Action",
+      "options" : {
+
+      }
+    }
+  ],
+  "defaultOptions" : {
+    "codeCoverage" : false
+  },
+  "testTargets" : [
+    {
+      "target" : {
+        "containerPath" : "container:",
+        "identifier" : "AuthenticationTests",
+        "name" : "AuthenticationTests"
+      }
+    }
```

Comment:
Set up a testplan for authentication tests so you can run them when you work in the AuthenticationHandler module